k8s/sonarqube/chart/sonarqube/templates/application.yaml

```diff
@@ -46,8 +46,8 @@ spec:
   componentKinds:
   - group: apps/v1
     kind: Deployment
-  - group: v1
-    kind: PersistentVolumeClaim
```

Comment:
Don't we longer want to display `PersistentVolumeClaim` in the UI?

Reply:
Let's keep the PVC here.